tests/Agent/IntegrationTests/IntegrationTests/RemoteServiceFixtures/AwsLambda/LambdaKinesisEventTriggerFixture.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+namespace NewRelic.Agent.IntegrationTests.RemoteServiceFixtures.AwsLambda
+{
+    public abstract class LambdaKinesisEventTriggerFixtureBase : LambdaSelfExecutingAssemblyFixture
+    {
+        private static string GetEventName(bool usesTimeWindow) => $"Kinesis{(usesTimeWindow ? "TimeWindow" : "")}Event";
+
+        protected LambdaKinesisEventTriggerFixtureBase(string targetFramework, bool isAsync, bool usesTimeWindow) :
+            base(targetFramework,
+                null,
+                $"LambdaSelfExecutingAssembly::LambdaSelfExecutingAssembly.Program::{GetEventName(usesTimeWindow)}Handler" + (isAsync ? "Async" : ""),
+                GetEventName(usesTimeWindow) + (isAsync ? "Async" : ""),
+                null)
+        {
+        }
+
+        // This event json works for both the normal Kinesis and the KinesisTimeWindowEvent.
+        // If we need data specifically defined in the KinesisTimeWindowEvent we can update the tests to separate
+        // the two events.
+        public void EnqueueEvent()
+        {
+            var eventJson = @"{
```

Comment:
Did you want to change these to be raw string literals?

Reply:
I can make that change with the next PR that I submit.